region/maridia/inner-pink.json

```diff
@@ -1732,6 +1787,39 @@
                     "Morph",
                     "canUseFrozenEnemies"
                   ]
+                },
+                {
+                  "name": "Cross Room Jump with Speedbooster",
+                  "notable": false,
+                  "requires": [
+                    "Morph",
+                    "canCrossRoomJumpIntoWater",
+                    "SpeedBooster",
+                    {"adjacentRunway": {
+                      "fromNode": 1,
+                      "usedTiles": 44,
+                      "inRoomPath": [ 1, 5 ],
+                      "physics": ["normal"]
+                    }}
+                  ],
+                  "note": "Needs max horizontal speed coming from 44 runway tiles in the adjacent room."
+                },
+                {
+                  "name": "Cross Room Jump with Speedbooster, Spring Ball",
+                  "notable": false,
+                  "requires": [
+                    "Morph",
+                    "canCrossRoomJumpIntoWater",
+                    "SpeedBooster",
+                    "SpringBall",
```

Comment:
`canSpringBallJump` 
and the tech needs a different name 🐸 Maybe springball bounce
Or to not be a tech

Reply:
I've been wanting to rename that one to springball bounce for a while. It's a tech in that it's used with speed, so you kind of have to know how to use it. But I think `canSpringBallJumpMidAir` would be better named `canSpringBallJump`